tamr_client/operation.py

```diff
@@ -19,6 +19,26 @@ class NotFound(TamrClientException):
     pass
 
 
+class Failed(TamrClientException):
+    """Raised when checking a failed operation.
+    """
+
+    pass
+
+
+def check(operation: Operation):
+    """Waits for the operation to finish and raises an exception if the operation was not successful.
+
+    Raises:
+        Failed: If the operation failed.
+    """
+    op = wait(operation)
```

Comment:
```suggestion
def check(session: Session, operation: Operation):
    """Waits for the operation to finish and raises an exception if the operation was not successful.

    Raises:
        Failed: If the operation failed.
    """
    op = wait(session, operation)
```
Needs a `session` now I think.

Reply:
Fixed!